### ISSUE TITLE
[3.4.0] Reports warnings about default exports

### ISSUE THREAD

**Issue by roblav96** [webpack/webpack#5374]
**What is the current behavior?**
Since updating to webpack@3.4.0 the compiler shows warnings about `export 'default'`.

**If the current behavior is a bug, please provide the steps to reproduce.**
```typescript
import Vue from 'vue'
import VueRouter from 'vue-router'
import Vuex from 'vuex'
import Vuetify from 'vuetify'
```

**What is the expected behavior?**
No warnings.

**Please mention other relevant information such as the browser version, Node.js version, webpack version and Operating System.**
```
WARNING in ./src/main.ts
18:8-17 "export 'default' (imported as 'VueRouter') was not found in 'vue-router'

WARNING in ./src/main.ts
19:8-12 "export 'default' (imported as 'Vuex') was not found in 'vuex'

WARNING in ./src/router.ts
42:24-33 "export 'default' (imported as 'VueRouter') was not found in 'vue-router'

WARNING in ./src/services/store.ts
8:23-27 "export 'default' (imported as 'Vuex') was not found in 'vuex'
webpack: Compiled with warnings.
```

Tried setting
```typescript
devServer: {
	clientLogLevel: 'error',
},
```
but still reported these warnings.
[vuejs.org#Recommended-Configuration](https://vuejs.org/v2/guide/typescript.html#Recommended-Configuration)

**Comment by bdwain** [comment]
i'm seeing this too. It only happens to me on rebuilds though (with webpack-dev-server). not the initial one. it first started when i went from webpack 3.3 to 3.4. everything still works, but i get lots of warnings. it's not just default exports though. "export 'connect' was not found in 'react-redux'

**Comment by dlong500** [comment]
I'm seeing it in 3.4.0 only on rebuilds as well with `webpack-dev-middleware`.  If I have babel transpile the es6 modules by disabling the `modules: false` directive then I don't see the warnings with 3.4.0 (but obviously that prevents tree shaking).

**Comment by sokra** [comment]
fixed in 3.4.1

**Comment by bdwain** [comment]
Thanks for fixing that so quickly!
